integration/cypress/integration/committee-detail-autoupdate.spec.js

```diff
@@ -8,7 +8,7 @@ describe("Get autoupdates for committees detail view", () => {
     const committeeData = {
       organization_id: 1,
       name: committeeName,
-      manager_ids: [1],
+      user_$can_manage_management_level: [1],
```

Comment:
is this a typo? (missing '_')
`user_$_can_manage_management_level: [1]`

Reply:
no, that was correct!
But there are 2 ways to use this template fields:
In payload it is `template_name`: {`replacement`: [list of ids]}
In code this is separated into 2 fields: `template_name`: [list of used replacements], `pre_$replacement_after`: [list of ids]
template_name: user_$_management_level
replacement: can_manage
pre:user_$
after:_management_level
Hope it's usefull:-)